components/Mask/src/index.svelte

```diff
@@ -83,8 +84,8 @@
 		style:left="{maskLeft}px"
 		style:width={maskWidth}
 		style:height={maskHeight}
-		style={color ? `background-color: ${color}` : ''}
-		class="k-mask--base {cnames}"
+		style={color ? `background-color: ${color}` : undefined}
```

Comment:
Please pay attention when using empty strings `''` which will lead to make empty style attribute, instead we should use undefined for all attrs.